exoctk/atmospheric_retrievals/examples.py

```diff
@@ -0,0 +1,319 @@
+#! /usr/bin/env python
+
+"""This module provides examples of how to use the ``platon_wrapper``
+module in ExoCTK's ``atmopsheric_retrieval`` subpackage.  Each example
+can be run using either the ``multinest`` method or the ``emcee``
```

Comment:
Another great question that can better be answered by a domain expert.  I think we are fine here though, since I refer them in the documentation to go see PLATON's documentation for more information.   As such, I think we are depending on PLATON to explain their methods, and so we don't have to worry about it.

Reply:
Sounds good to me! I figured these sorts of things would be covered somewhere but wasn't sure if it should be here, but I think you're right that by providing the link to the code you're wrapping then people have the resources they need.